File: src/main/java/com/chrisrm/idea/themes/literature/club/MTYuriTheme.java
package com.chrisrm.idea.themes.literature.club;

public final class MTYuriTheme extends MTDokiDokiTheme {
  public MTYuriTheme() {
    super("mt.yuri", "Material Yuri", false, "Yuri");
  }

  @Override
  public String getBackgroundColorString() {
    return "e8e4ff";
  }

  @Override
  public String getSecondaryBackgroundColorString() {
    return "bbb8ff";
  }

  @Override
  public String getSecondaryForegroundColorString() {
    return "562474";
  }

  @Override
  public String getSelectionForegroundColorString() {
    return "c06fff";
  }

  @Override
  public String getSelectionBackgroundColorString() {
    return "e4caff";
  }

  @Override
  public String getTreeSelectionBackgroundColorString() {
    return "d3a5fa";
  }

  @Override
  public String getMenuBarSelectionForegroundColorString() {
    return "ffffff";
  }

  @Override
  public String getMenuBarSelectionBackgroundColorString() {
    return "7c0e9d";
  }

  @Override
  public String getNotificationsColorString() {
    return "d3ceff";
  }

  @Override
  public String getContrastColorString() {
    return "ccaaff";
  }

  @Override
  public String getEditorTabColorString() {
    return getContrastColorString();
  }
}

File: src/main/java/com/chrisrm/idea/themes/literature/club/MTYuriTheme.kt
package com.chrisrm.idea.themes.literature.club

class MTYuriTheme : MTDokiDokiTheme("mt.yuri", "Material Yuri", false, "Yuri") {

    override fun getBackgroundColorString(): String = "e8e4ff"

    override fun getClubMember(): String = "yuri.png"

    override fun getSecondaryBackgroundColorString(): String = "bbb8ff"

    override fun getSecondaryForegroundColorString(): String = "562474"

    override fun getSelectionForegroundColorString(): String = "c06fff"

    override fun getSelectionBackgroundColorString(): String = "e4caff"

    override fun getTreeSelectionBackgroundColorString(): String = "d3a5fa"

    override fun getMenuBarSelectionForegroundColorString(): String = "ffffff"

    override fun getMenuBarSelectionBackgroundColorString(): String = "7c0e9d"

    override fun getNotificationsColorString(): String = "d3ceff"

    override fun getContrastColorString(): String = "ccaaff"

    override fun getEditorTabColorString(): String = contrastColorString
}
